tests/integration/app_routes/admin/admin_routes/test_admin_jobs_routes.py

```
@@ -479,16 +479,20 @@ def test_delete_fix_nested_main_files_job(admin_jobs_client):
     assert len(store.list()) == 0
 
 
-def test_delete_nonexistent_job(admin_jobs_client):
+def test_delete_nonexistent_job(monkeypatch, admin_jobs_client):
     """Test deleting a non-existent job shows appropriate error."""
+    mock_flash = Mock()
+    monkeypatch.setattr("src.main_app.app_routes.admin_routes.jobs.flash", mock_flash)
+
     client, store = admin_jobs_client
```

Ruff (0.15.12)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)